### ISSUE TITLE
Cpanel installation

### ISSUE THREAD

**Issue by winklevos** [dazzletheweb/mautic-sendinblue#3]
i tried installing this plugin into the /plugin folder on my cpanel host, on the latest version. I could see the plugin but now it's errored the install, any way around this?

[07-May-2020 17:08:43] PHP Fatal error:  Declaration of MauticPlugin\MauticSendinblueBundle\Swiftmailer\Transport\SendinblueApiTransport::send(Swift_Mime_SimpleMessage $message, &$failedRecipients = NULL) must be compatible with Mautic\EmailBundle\Swiftmailer\Transport\AbstractTokenArrayTransport::send(Swift_Mime_Message $message, &$failedRecipients = NULL) in /public_html/m/plugins/MauticSendinblueBundle/Swiftmailer/Transport/SendinblueApiTransport.php on line 29

**Comment by ojacquet** [comment]
You will have to remove it. It is not possible to use this plugin without composer.

**Comment by winklevos** [comment]
hmm, alright then. Looks like it's broked my install [07-May-2020 07:35:46 UTC] Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: You have requested a non-existent service "swiftmailer.mailer.transport.mautic.transport.sendinblue_api". - in file /public_html/m/vendor/symfony/dependency-injection/ContainerBuilder.php - at line 817

**Comment by winklevos** [comment]
Edit: apologies my provider lead me the wrong way this is composer doing dependency calcs
Got to hosting provider to correct their composer install but all i get on install is 
`Fatal error: Allowed memory size of 1610612736 bytes exhausted (tried to allocate 4096 bytes) in phar:///opt/cpanel/composer/bin/composer/src/Composer/DependencyResolver/RuleSetGenerator.php on line 129` i'm no composer expert but why does installing your plugin try to allocate 1.6G of memory?

**Comment by winklevos** [comment]
@ojacquet got around the install issues and have installed into the mautic main dir now seeing this, with mautic not resolving at all

`[08-May-2020 04:04:25 UTC] PHP Fatal error:  Uncaught Error: Class 'Mautic\CoreBundle\ErrorHandler\ErrorHandler' not found in /home/winkfitc/public_html/m/index.php:29
Stack trace:
#0 {main}
  thrown in /home/winkfitc/public_html/m/index.php on line 29
`

**Comment by ojacquet** [comment]
Yes, that is because you also need the actual Mautic composer files which are not included in a default Mautic installation. Download them from the Mautic repository, but make sure you use the correct version https://github.com/mautic/mautic/tree/2.16.2 is compatible with version 1.0.1 of this plugin.

**Comment by winklevos** [comment]
thanks for the help @ojacquet grabbed the composer.json from mautic git and installed, all working now, thanks

**Comment by luiszxcv** [comment]
Olá estou com o mesmo problema aqui, como resolver?

> Editar: desculpas meu provedor me levou para o caminho errado, este é o compositor fazendo cálculos de dependência Tenho que o provedor de hospedagem corrigir a instalação do compositor, mas tudo o que recebo na instalação é `Fatal error: Allowed memory size of 1610612736 bytes exhausted (tried to allocate 4096 bytes) in phar:///opt/cpanel/composer/bin/composer/src/Composer/DependencyResolver/RuleSetGenerator.php on line 129`que não sou especialista em compositor, mas por que a instalação do seu plugin tenta alocar 1,6 G de memória?

**Comment by luiszxcv** [comment]
@winklevos
